### PR TITLE
Test added for scenario T-3904085: Modify Household Account Contact Address

### PR DESCRIPTION
**Pull request by salvadorsfdc**
Test added for scenario T-3904085: Using Household account model, modify an existing Contact Address and assess new address creation under Household account	

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: unpackaged/functional_tests/address/classes/Address_FTST.cls [hunk 1]
```diff
@@ -132,6 +132,26 @@ public with sharing class Address_FTST {
         return newContact;
     }
 
+    private static Contact createTestContactWithoutAddress(Id accountId) {
```
Comment (xinnanlisf) [inline unpackaged/functional_tests/address/classes/Address_FTST.cls:135]: Curious why not use util_testdata to create test contact? I thought we had a similar method.

File: unpackaged/functional_tests/address/classes/Address_FTST.cls [hunk 2]
```diff
@@ -290,4 +310,114 @@ public with sharing class Address_FTST {
             'The current address is incorrect'
         );
     }
+
+    /**
+     * @description T-3904085: Using Household account model, modify an existing Contact Address and assess new address
```
Comment (xinnanlisf) [inline unpackaged/functional_tests/address/classes/Address_FTST.cls:315]: Just a reminder to check the 'automate' checkbox for test scenario. Please ignore if you already did.

File: unpackaged/functional_tests/address/classes/Address_FTST.cls [hunk 3]
```diff
@@ -290,4 +310,114 @@ public with sharing class Address_FTST {
...
+     * creation under Household account
+     */
+    @isTest
+    private static void updateExistingHouseholdAccountContactAddress() {
+        hed__Hierarchy_Settings__c orgSettings = createSettings();
```
Comment (xinnanlisf) [inline unpackaged/functional_tests/address/classes/Address_FTST.cls:320]: I notice that this settings setup approach has been merged already but we do have some settings pattern in eda package. My understand is that you don't have to upsert the settings as it is static.

File: unpackaged/functional_tests/address/classes/Address_FTST.cls (hunk 2 above)
Comment (salvadorsfdc) [reply unpackaged/functional_tests/address/classes/Address_FTST.cls:315]: Thanks for reminding me, I just changed the execution type to automated.

File: unpackaged/functional_tests/address/classes/Address_FTST.cls (hunk 1 above)
Comment (salvadorsfdc) [reply unpackaged/functional_tests/address/classes/Address_FTST.cls:135]: you're right, I'll reference the UTIL_UnitTestData_API.getMultipleTestContacts in my method and just add the extra field mappings that I need. Thanks!

File: unpackaged/functional_tests/address/classes/Address_FTST.cls (hunk 3 above)
Comment (salvadorsfdc) [reply unpackaged/functional_tests/address/classes/Address_FTST.cls:280]: you are right, the upsert and update of orgSettings variable is not really needed. Just fixed it. Thanks!

**Review by xinnanlisf — APPROVED** [review]

**Comment by xinnanlisf** [comment]
Functional tests passed.

**Pull request merged**